pallets/transaction-payment/src/lib.rs

```diff
@@ -392,21 +464,23 @@ where
         info: &DispatchInfoOf<T::Call>,
         len: usize,
     ) -> Result<(BalanceOf<T>, Option<NegativeImbalanceOf<T>>), TransactionValidityError> {
-        let fee = Module::<T>::compute_fee(len as u32, info, 0u32.into());
+        let tip = self.0;
```

Comment:
@satyamakgec So AIUI, we want to revert all the changes in this function?

Reply:
Yes in this function and some code comments which have mention of a tip. and could you also please add the tip change in the top of the pallet.